src/js/views/PeaksList/PeaksList.module.css

```diff
@@ -5,7 +5,14 @@
 
 .peaksList {
     margin: 1%;
-    margin-top: 6%;
     width: 100%;
     text-align: center;
 }
+/* 
+.listLeft {
+  margin-right: 1%;
+}
+
+.listRight {
+  margin-left: 1%;
+} */
```

Comment:
To jest do usunięcia czy do odkomentowania? Bo jest używane w `PeaksList.tsx`

Reply:
To jest kod który przywraca wygląd PeaksListy który się delikatnie zmienił. Dorzucę go, a Ty możesz sprawdzić czy pasuje Ci nowy wygląd PeaksListy

Reply:
Nowa lista jest szersza od starej, ale bez problemu można to naprawić określając % szerokości.

Reply:
Ok, może być szersza